Guacamole Service properties documentation (#1380)
microsoft/AzureTRE: docs/tre-templates/workspace-services/guacamole.md

  make bundle-build DIR=./templates/workspace_services/guacamole
  make bundle-install DIR=./templates/workspace_services/guacamole
  ```

## Guacamole Workspace Service Configuration

> On deploying a Guacamole service into a workspace the following properties are exposed.

Reviewer: ```suggestion
When deploying a Guacamole service into a workspace the following properties need to be configured.
```

Reviewer: maybe mention that the last property is mandatory and the rest are optional?
you can list the mandatory property first.

Author: @eladiw good point.  Will need doing separate to this PR but currently the Client ID is not set as a required property in the template_schema.json file for the Guacamole Service despite having no default.

Reviewer: Even if its not marked as mandatory (wrongfully) I guess the doc can still be correct. wdyt

Author: Agree. Whether the schema highlights it or not it is still mandatory to successfully deploy the workspace service

Author: All suggested changes implemented
